source/System.h

```diff
@@ -95,6 +95,8 @@ class System {
 	
 	// Get a list of systems you can travel to through hyperspace from here.
 	const std::set<const System *> &Links() const;
+	// Additional travel distance to target for ships entering through hyperspace.
+	double ArrivalDistance() const;
```

Comment:
To me, the function name does not match the description. I would expect that something called `ArrivalDistance` would be expressing an absolute distance, not an offset. A name like `AdditionalArrivalDistance` or `ArrivalOffset` might be clearer IMO.

Reply:
Changed to `ExtraArrivalDistance`.